src/Cake.Common/Tools/Chocolatey/ChocolateyAliases.cs

```diff
@@ -0,0 +1,830 @@
+        ///                                     Owners                  = new[] {"Contoso"},
+        ///                                     Summary                 = "Excellent summary of what the package does",
+        ///                                     Description             = "The description of the package",
+        ///                                     ProjectUrl              = new Uri("https://github.com/SomeUser/TestChocolatey/"),
+        ///                                     PackageSourceUrl        = new Uri("https://github.com/SomeUser/TestChocolatey/"),
+        ///                                     ProjectSourceUrl        = new Uri("https://github.com/SomeUser/TestChocolatey/"),
+        ///                                     DocsUrl                 = new Uri("https://github.com/SomeUser/TestChocolatey/"),
+        ///                                     MailingListUrl          = new Uri("https://github.com/SomeUser/TestChocolatey/"),
+        ///                                     BugTrackerUrl           = new Uri("https://github.com/SomeUser/TestChocolatey/"),
+        ///                                     Tags                    = new [] {"Cake", "Script", "Build"},
+        ///                                     Copyright               = "Some company 2015",
+        ///                                     LicenseUrl              = new Uri("https://github.com/SomeUser/TestChocolatey/blob/master/LICENSE.md"),
+        ///                                     RequireLicenseAcceptance= false,
+        ///                                     IconUrl                 = new Uri("http://cdn.rawgit.com/SomeUser/TestChocolatey/master/icons/testchocolatey.png"),
+        ///                                     ReleaseNotes            = new [] {"Bug fixes", "Issue fixes", "Typos"},
+        ///                                     Files                   = new [] {
+        ///                                                                          new ChocolateyNuSpecContent {Source = "bin/TestChocolatey.dll", Target = "bin"},
+        ///                                                                       },
+        ///                                     Debug                   = false,
+        ///                                     Verbose                 = false,
+        ///                                     Force                   = false,
+        ///                                     Noop                    = false,
+        ///                                     LimitOutput             = false,
+        ///                                     ExecutionTimeout        = 13,
+        ///                                     CacheLocation           = @"C:\temp",
+        ///                                     AllowUnoffical          = false
+        ///                                 };
+        ///
+        ///     ChocolateyPack(chocolateyPackSettings);
```

Comment:
Cool :-)

Where should this dependency be noted?

Sent from my Windows Phone
________________________________
From: Mattias Karlsson<mailto:notifications@github.com>
Sent: ‎11/‎10/‎2015 18:15
To: cake-build/cake<mailto:cake@noreply.github.com>
Cc: Gary Ewan Park<mailto:gep13@gep13.co.uk>
Subject: Re: [cake] [WIP] Chocolatey Addin for Cake (#434)

> +        ///                                     IconUrl                 = new Uri("http://cdn.rawgit.com/SomeUser/TestChocolatey/master/icons/testchocolatey.png"),
> +        ///                                     ReleaseNotes            = new [] {"Bug fixes", "Issue fixes", "Typos"},
> +        ///                                     Files                   = new [] {
> +        ///                                                                          new ChocolateyNuSpecContent {Source = "bin/TestChocolatey.dll", Target = "bin"},
> +        ///                                                                       },
> +        ///                                     Debug                   = false,
> +        ///                                     Verbose                 = false,
> +        ///                                     Force                   = false,
> +        ///                                     Noop                    = false,
> +        ///                                     LimitOutput             = false,
> +        ///                                     ExecutionTimeout        = 13,
> +        ///                                     CacheLocation           = @"C:\temp",
> +        ///                                     AllowUnoffical          = false
> +        ///                                 };
> +        ///
> +        ///     ChocolateyPack(chocolateyPackSettings);

Ah was on `Chocolatey v0.9.9.1` upgrading to `Chocolatey v0.9.9.11` and now it works.

---
Reply to this email directly or view it on GitHub:
https://github.com/cake-build/cake/pull/434/files#r41712943